### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by talrejah02**
please review, suggestion's are welcomed.
ps: not responsive yet

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -43,14 +37,14 @@ body {
...
-  width: 15rem;
+  width: 16rem;
 }
 .nav-link {
   font-size: 19px;
   text-decoration: none;
   border-radius: 10px;
   color: var(--font-primary);
-  padding: 0.2rem 0.9rem 0.2rem 0.9rem;
+  padding: 0.5rem 0.9rem 0.5rem 0.9rem;
```
Comment (kshitijthatte) [inline style.css]: You can use the shorthand property for padding (two values)

File: components/Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
+@import url(../../docs-css/reset.css);
+@import url(../common-css/drawer.css);
+@import url(../common-css/container.css);
+@import url(../../style.css);
+
+body{
+    background-color: var(--secondary);
+    margin: 0;   
+}
```
Comment (kshitijthatte) [inline components/Alert/alert.css:9]: Avoid styling the body directly.

File: style.css [hunk 3]
```diff
@@ -43,14 +37,14 @@ body {
...
   flex-wrap: wrap;
   justify-content: space-evenly;
-  width: 15rem;
+  width: 16rem;
 }
 .nav-link {
   font-size: 19px;
   text-decoration: none;
   border-radius: 10px;
   color: var(--font-primary);
```
Comment (kshitijthatte) [inline style.css]: The contrast ratio is pretty bad at 2.9 with this color on the blue nav background.

**Review by kshitijthatte — APPROVED** [review]

File: components/common-css/container.css [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+ margin-bottom: 0.2rem;
+}
+.code pre{
+    height: 100%;
+    width: 100%;
+    margin:0;
+}
+code{
+margin: 1rem;
+margin-top: 0;
```
Comment (SaurabhChawda) [inline components/common-css/container.css]: You can use Shorthand property for margin top and bottom.

**Review by SaurabhChawda — APPROVED** [review]

**Comment by SaurabhChawda** [comment]
You have used an image of 1.8MB, You should reduce the size of the Image.

**Pull request merged**